src/config.rs

```diff
@@ -492,6 +496,7 @@ pub fn construct_config(pc: ProxyConfig) -> Result<Config, Error> {
         inpod_port_reuse: parse_default(INPOD_PORT_REUSE, true)?,
         inpod_mark: parse_default(INPOD_MARK, DEFAULT_INPOD_MARK)?,
         fake_self_inbound: false,
+        testing_dedicated_mark: 0,
```

Comment:
I am relatively sure that even non-testing usages of `dedicated` mode might need this mark, so TBH I am fine just making it `dedicated_mark` and setting it either via env or via `DEFAULT_DEDICATED_MARK` like we do with inpod above.

Alternatively - we just rename `inpod_mark` to `proxy_mark`, default to `1337` (current inpod default + also the current sidecar/envoy default) and use the same config val under both modes.


tl;dr I think this will be necessary for `dedicated` mode outside of just tests so we might as well formalize it? thoughts?

Reply:
I was on the fence, went with test only since I wasn't sure a concrete use case where running as a different UID/GID wasn't viable. Not opposed if there is (or if we think its simpler to just make it a real thing -- with the inpod_mark collapsing it may just be quite a bit simpler to do so)

Reply:
ok, I made the change